.github/workflows/e2e.yml

```diff
@@ -0,0 +1,53 @@
+name: e2e
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
```

Comment:
It seems like both jobs are the same, but the browser. How about the matrix strategy, like we used in the build job?

Reply:
Can I use the matrix parameters just in `with` statements?

Or should this also work?

```
end2end:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        browser: ['e2e:chrome', 'e2e:firefox']
    name: e2e Test Chrome
    steps:
      ...
      - name: Run cypress tests
        run: yarn ${{ matrix.browser }}
```

Reply:
you can use it everywhere. (please remember changing the job names, too)

Reply:
done